### PR TITLE
added type definitions for libsodium-wrappers crypto_aead_aegis functions and constants

### PR DESCRIPTION
**Pull request by bschick**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change. -> Tests a very minimal, did not expand. tested in an seperate project
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `pnpm test <package to test>`]  -> hitting  ENFILE: file table overflow, scandir

Select one of these and delete the others:

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/jedisct1/libsodium.js/tree/master/wrapper/symbols
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`  ->  looks invalid already... didn't change

### CODE REVIEW

File: types/libsodium-wrappers/index.d.ts [hunk 1]
```diff
@@ -387,6 +397,152 @@ export function crypto_aead_xchacha20poly1305_ietf_encrypt_detached(
...
+
+export function crypto_aead_aegis256_encrypt(
+    message: string | Uint8Array,
+    additional_data: string | Uint8Array | null,
+    secret_nonce: string | Uint8Array | null,
+    public_nonce: Uint8Array,
+    key: Uint8Array,
+    outputFormat?: Uint8ArrayOutputFormat | null,
+): Uint8Array;
+export function crypto_aead_aegis256_encrypt(
```
Comment (ffflorian) [inline types/libsodium-wrappers/index.d.ts:517]:
```suggestion

export function crypto_aead_aegis256_encrypt(
```

File: types/libsodium-wrappers/index.d.ts [hunk 2]
```diff
@@ -387,6 +397,152 @@ export function crypto_aead_xchacha20poly1305_ietf_encrypt_detached(
...
+export function crypto_aead_aegis256_decrypt_detached(
+    secret_nonce: string | Uint8Array | null,
+    ciphertext: string | Uint8Array,
+    mac: Uint8Array,
+    additional_data: string | Uint8Array | null,
+    public_nonce: Uint8Array,
+    key: Uint8Array,
+    outputFormat?: Uint8ArrayOutputFormat | null,
+): Uint8Array;
+export function crypto_aead_aegis256_decrypt_detached(
```
Comment (ffflorian) [inline types/libsodium-wrappers/index.d.ts:499]:
```suggestion

export function crypto_aead_aegis256_decrypt_detached(
```

File: types/libsodium-wrappers/index.d.ts [hunk 3]
```diff
@@ -387,6 +397,152 @@ export function crypto_aead_xchacha20poly1305_ietf_encrypt_detached(
...
+
+export function crypto_aead_aegis256_encrypt_detached(
+    message: string | Uint8Array,
+    additional_data: string | Uint8Array | null,
+    secret_nonce: string | Uint8Array | null,
+    public_nonce: Uint8Array,
+    key: Uint8Array,
+    outputFormat?: Uint8ArrayOutputFormat | null,
+): CryptoBox;
+export function crypto_aead_aegis256_encrypt_detached(
```
Comment (ffflorian) [inline types/libsodium-wrappers/index.d.ts:534]:
```suggestion

export function crypto_aead_aegis256_encrypt_detached(
```

**Review by ffflorian — CHANGES_REQUESTED** [review]

**Review by ffflorian — APPROVED** [review]
Thanks!

**Comment by bschick** [comment]
Ready to merge

**Pull request merged**